src/jsMain/js/FakeClientDevice.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react';
```

Comment:
This is super super nit-picky and really of no value, but I _highly_ recommend installing the Prettier plugin (https://plugins.jetbrains.com/plugin/10456-prettier) and running it before committing  any JS.

It automatically formats your JS/react code for you like IDE's usually do, and it makes it so no one ever has to comment about a missing space or semi-colon in your code.